prebuilt_forms/dynamic_location.php

```diff
@@ -556,5 +565,22 @@ protected static function getSubmitButtons($args) {
     return $r;
   }
 
+  private function userOwnsLocation() {
```

Comment:
Coding standards - function comment header missing.

Reply:
Also renamed function to better reflect what it does.